.github/workflows/deployProd.yml

```diff
@@ -28,6 +28,7 @@ jobs:
 
   build_frontend:
     runs-on: ubuntu-latest
+    environment: Production
```

Comment:
Do we need this? 🤔

Reply:
this is how GitHub knows to grab the production-specific variables since [this variable is scoped to just the prod env](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-variables)

we do [something similar for the dev deploy actions](https://github.com/CDCgov/prime-simplereport/blob/aa25fb9e5a3994b420a1cb7de0de6a3eb8f0b958/.github/workflows/deployDev.yml#L43)

Reply:
thank you for the explanation!